Shared/Flows/Onboarding/TextInputViewController.swift

```diff
@@ -73,12 +73,19 @@ class TextInputViewController<ResultType>: ViewController, Sizeable, Completable
                                            for: UIControl.Event.editingChanged)
         self.textEntry.textField.delegate = self
 
-        KeyboardManager.shared.addKeyboardObservers(with: nil)
```

Comment:
I changed the keyboard manager so you don't have to specify the current input accessory. Instead, it automatically detects it.